Update agent token docs to point at tokens page (#1790)
buildkite/docs: pages/agent/v3/tokens.md

The Buildkite Agent requires an agent token to connect to Buildkite and register for work. If you are an admin of your Buildkite organization, you can view the tokens on your [Agents page](https://buildkite.com/organizations/-/agents).

## Finding and creating tokens

When you create a new organization in Buildkite, a default agent token is created. This token can be used for testing and development, but it's recommended to create new, specific tokens for each new environment.

> Tokens can be found and managed on your [Agent Tokens page](https://buildkite.com/organizations/-/agent-tokens).

Reviewer: Let's use the active voice here:

```suggestion
You can find and manage your tokens on the [Agent Tokens page](https://buildkite.com/organizations/-/agent-tokens).
```

Reviewer: Is it expected that I get a 404 when I go to this URL?

Author: Yeah there's no central "agent tokens" page any more, it's within the [Clusters](https://buildkite.com/organizations/-/clusters) area. This docs page probably needs a little rework to point at the area and how to find tokens within.

> Tokens can also be created using [the GraphQL API](/docs/apis/graphql/schemas/mutation/agenttokencreate).

Reviewer: Active voice and remove the article from the link text:

```suggestion
You can also create tokens using the [GraphQL API](/docs/apis/graphql/schemas/mutation/agenttokencreate).
```

## Using and storing tokens

The token is used by the Buildkite Agent's [start](/docs/agent/v3/cli-start#starting-an-agent) command, and can be provided on the command line, set in the [configuration file](/docs/agent/v3/configuration), or provided using the [environment variable](/docs/pipelines/environment-variables) `BUILDKITE_AGENT_TOKEN`.

It's recommended you use your platform's secret storage (such as the [AWS Systems Manager Parameter Store](https://docs.aws.amazon.com/systems-manager/latest/userguide/systems-manager-paramstore.html)) to allow for easier rollover and management of your agent tokens.

## Revoking tokens

> Tokens can be revoked from your [Agent Tokens page](https://buildkite.com/organizations/-/agent-tokens). Once a token is revoked, no new agents will be able to start with that token. Revoking a token does not affect any connected agents.

Reviewer: Active voice + tightening:

```suggestion
You can revoke tokens from the [Agent Tokens page](https://buildkite.com/organizations/-/agent-tokens). Once you revoke a token, no new agents can start with that token. Revoking a token does not affect any connected agents.
```

> Tokens can also be revoked using [the GraphQL API](/docs/apis/graphql/schemas/mutation/agenttokenrevoke).

Reviewer: Active voice + link text:

```suggestion
You can also revoke tokens using the [GraphQL API](/docs/apis/graphql/schemas/mutation/agenttokenrevoke).
```
